frontend/src/components/files/ListingItem.vue

```diff
@@ -106,6 +106,9 @@ export default {
       return filesize(this.size);
     },
     humanTime: function () {
+      if (this.user.dateFormat) {
+        return moment(this.modified).format("YYYY-MM-DD HH:mm");
```

Comment:
```suggestion
        return moment(this.modified).format("L LT");
```
FileBrowser supports multiples languages, dates should be displayed with a localized format.